plugins/SegmentEditor/SegmentEditor.php

```diff
@@ -61,8 +61,13 @@ public function addSystemSummaryItems(&$systemSummary)
 
     function getSegmentEditorHtml(&$out)
     {
-        $selector = new SegmentSelectorControl();
-        $out .= $selector->render();
+        try {
+            $selector = new SegmentSelectorControl();
+            $out .= $selector->render();
+        } catch (\Exception $ex) {
+            print $ex->getTraceAsString()."\n";
```

Comment:
guess would be better to log the error instead of printing it...

Reply:
sorry, that's was for debugging something else, accidentally committed it

Reply:
btw this still needs to be removed right?

Reply:
@diosmosis this still needs to be removed, be good to not forget.